### PR TITLE
Milestone2 has been created and all components were created

### PR DESCRIPTION
**Pull request by cochabambinoski**
All components were created according to the document of requirements.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { value } = this.props;
```
Comment (suretrust) [inline src/components/Button.js]: `name` seems like a better description for the button's name.
```suggestion
    const { name } = this.props;
```

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    return (
+      <button className={classStyle} type="submit">
+        {value}
+      </button>
+    );
+  }
+}
+
+Button.propTypes = {
+  value: PropTypes.string,
```
Comment (suretrust) [inline src/components/Button.js]: This should be `required`. Without the `name` props, the button is useless, isn't it?
```suggestion
  value: PropTypes.string.required,
```

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
```
Comment (suretrust) [inline src/components/Button.js:11]: I'm not sure you need this at all here.
```suggestion

```

File: src/components/Display.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import PropTypes from 'prop-types';
+import '../styles/Display.css';
+
+class Display extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
```
Comment (suretrust) [inline src/components/Display.js:11]: We don't need this for now too, right?
```suggestion
```

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @cochabambinoski, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Comment by cochabambinoski** [comment]
all those changes look like unnecessary... they are more suggestions and teaching observations, non-necessary to keep moving on, I mean, renaming a const, sounds more a self taste like. 

I can not just delete the constructor, props, and states like this, because even if it works, it's not accepted within the linter... To make it work I would have to create Functional Components instead... like this:

Function Button(props){
    return ()
}

Changing normal Components to Functional Components also is not worth it... I am going to use states and props later for sure, so I am going to delete them later to go back to what I already deleted in the first instance...

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: :clap: 
Hi @cochabambinoski :hand: 
Nice job on this milestone :clap: 
Good luck with the next one :smile: 

Happy Coding :computer: 
**Sergio Torres**

**Pull request merged**